attestor/command/check.go

```diff
@@ -84,10 +84,10 @@ func RunCheckWithParams(repoURL, commitSHA, policyPath string) (policy.PolicyRes
 	}
 
 	requiredChecks := attestationPolicy.GetRequiredChecksForPolicy()
-
+	b := checks.BinaryArtifactsCheck{}
 	enabledChecks := map[string]checker.Check{
 		checks.CheckBinaryArtifacts: {
-			Fn: checks.BinaryArtifacts,
+			Fn: b.Run,
```

Comment:
As we're leaving `checks.BinaryArtifacts`, might as well leave it here for readability.

Reply:
👍